core/src/components/select-option/usage/stencil.md

```diff
@@ -0,0 +1,159 @@
+        <ion-label>Select: Alert Interface</ion-label>
+        <ion-select interfaceOptions={options}>
+          <ion-select-option value="brown">Brown</ion-select-option>
+          <ion-select-option value="blonde">Blonde</ion-select-option>
+          <ion-select-option value="black">Black</ion-select-option>
+          <ion-select-option value="red">Red</ion-select-option>
+        </ion-select>
+      </ion-item>,
+
+      <ion-item>
+        <ion-label>Select: Alert Interface (Multiple Selection)</ion-label>
+        <ion-select interfaceOptions={options} multiple={true}>
+          <ion-select-option value="brown">Brown</ion-select-option>
+          <ion-select-option value="blonde">Blonde</ion-select-option>
+          <ion-select-option value="black">Black</ion-select-option>
+          <ion-select-option value="red">Red</ion-select-option>
+        </ion-select>
+      </ion-item>,
+
+      <ion-item>
+        <ion-label>Select: Popover Interface</ion-label>
+        <ion-select interface="popover" interfaceOptions={options}>
+          <ion-select-option value="brown">Brown</ion-select-option>
+          <ion-select-option value="blonde">Blonde</ion-select-option>
+          <ion-select-option value="black">Black</ion-select-option>
+          <ion-select-option value="red">Red</ion-select-option>
+        </ion-select>
+      </ion-item>,
+
+      <ion-item>
+        <ion-label>Select: Action Sheet Interface</ion-label>
+        <ion-select interface="action-sheet" interfaceOptions={options}>
+          <ion-select-option value="brown">Brown</ion-select-option>
+          <ion-select-option value="blonde">Blonde</ion-select-option>
+          <ion-select-option value="black">Black</ion-select-option>
+          <ion-select-option value="red">Red</ion-select-option>
+        </ion-select>
+      </ion-item>
+    ];
+  }
+}
+```
+
+```css
+/* Popover Interface: set color for the popover using Item's CSS variables */
+.my-custom-interface .select-interface-option {
+  --color: #971e49;
+  --color-hover: #79193b;
+}
+
+/* Action Sheet Interface: set color for the action sheet using its button CSS variables */
+.my-custom-interface .select-interface-option {
+  --button-color: #971e49;
+  --button-color-hover: #79193b;
+}
+
+/* Alert Interface: set color for alert options (single selection) */
+.my-custom-interface .select-interface-option .alert-radio-label {
+  color: #971e49;
+}
+
+/* Alert Interface: set color for alert options (multiple selection) */
+.my-custom-interface .select-interface-option .alert-checkbox-label {
+  color: #971e49;
+}
+
+/* Alert Interface: set color for checked alert options (single selection) */
+.my-custom-interface .select-interface-option[aria-checked=true] .alert-radio-label {
+  color: #79193b;
+}
+
+/* Alert Interface: set color for checked alert options (multiple selection) */
+.my-custom-interface .select-interface-option[aria-checked=true] .alert-checkbox-label {
+  color: #79193b;
+}
+```
```

Comment:
For these it would be cool if users could visualize it similar to how MDN does it: https://developer.mozilla.org/en-US/docs/Web/CSS/filter.

The code by itself is good technical information, but I'm not sure people would know what it looks like in their head.

Reply:
What do you mean? Like the code at the top that changes the image?

Reply:
Yeah like if we had a live demo of an open select, and you would click different options to see the buttons styles differently.